content/legal/privacy-notice.md

```diff
@@ -302,50 +299,48 @@ We never store password data in plain text; instead they are stored hashed (with
 
-In encrypted rooms, the data is stored in our databases but the encryption keys are stored only on your devices or by yourself. Users can optionally backup an encrypted copy of their keys on the Service to aid recovery if they lose all their keys and devices. This key backup is encrypted by a recovery key that only the user has access to. Element (the company, employees, and contractors) are unable to read your message content in our database. If you lose access to your encryption keys, you lose access to your messages forever.
+In encrypted rooms, the data is stored in our databases but the encryption keys are stored only on your devices or by yourself. Users can optionally backup an encrypted copy of their keys on the Service to aid recovery if they lose all their keys and devices. This key backup is encrypted by a recovery key that only the user has access to. [Element](https://element.io) (the company, including its employees and contractors) are unable to read your message content in our database. If you lose access to your encryption keys, you lose access to your messages forever.
 
 We use HTTPS to transfer all data. End-to-end encrypted messaging data is stored encrypted using AES-256, using message keys generated using the Olm and Megolm cryptographic ratchets.
 
-### 2.10 What are the guidelines matrix.org follows when accessing my data?
+### 2.10 What are the guidelines the Foundation follows when accessing my data?
 
-As per 2.11, the Foundation contracts Element to manage the hosting and data management of the homeserver. We restrict who at Element (employees and contractors) can access user data to roles which require access in order to maintain the health of the Service;
+As per 2.11, the Foundation contracts [Element](https://element.io) to manage the hosting and data management of the homeserver. We restrict who at [Element](https://element.io) (employees and contractors) can access user data to roles which require access in order to maintain the health of the Service;
 
 We have technical procedures in place to prevent unauthorised access to user data;
 
 We never share what we see with other users or the general public.
 
 ### 2.11 Who else has access to my data?
 
-Element is a Processor of your data, managing the homeserver on behalf of the Matrix.org Foundation. We host the majority of the Service in Mythic Beasts data centres. Here’s Mythic Beast’s [privacy policy](https://www.mythic-beasts.com/terms/privacy).
+[Element](https://element.io) is a Processor of your data, managing the homeserver on behalf of the Foundation. We host the majority of the Service in Mythic Beasts data centres. Here’s Mythic Beast’s [privacy policy](https://www.mythic-beasts.com/terms/privacy).
```

Comment:
For the next iteration: The implication from the following sentence is: Does Mythic Beasts not control physical access to their data centres?